src/BuiltInTools/dotnet-format/Utilities/AssemblyResolver.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the MIT license.  See License.txt in the project root for license information.
+
+using System.Reflection;
+using System.Runtime.Loader;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.CodeAnalysis.Tools.Utilities
+{
+    internal static class AssemblyResolver
+    {
+        private static readonly string[] s_extensions = new[] { "ni.dll", "ni.exe", "dll", "exe" };
```

Comment:
What are ni.dll/ni.exe?

Reply:
I don't see this file is used. @JoeRobich  might know more, it might just be some dead code.

Reply:
I believe is stands for native image. I agree that this is code is no longer used and can be removed.